packaging/test_plugin_indexing.py

```diff
@@ -932,6 +932,29 @@ def logical_filesystem_for_indexing(self,objects,session):
             for p in collections_to_delete:
                 session.assert_icommand(['irm', '-rf', p],'STDOUT','')
 
+    def test_indexing_with_atomic_acl_ops__81(self):
+        with indexing_plugin__installed():
+            test_coll = self.test_coll_name()
+            INDEX = DEFAULT_METADATA_INDEX + "__81_{:%s.%f}".format(_datetime.now())
+            create_metadata_index (INDEX)
+            try:
+                with session.make_session_for_existing_admin() as admin_session:
+                    admin_session.assert_icommand("""imeta set -C {0} irods::indexing::index {1}::metadata elasticsearch""".format(test_coll, INDEX))
+                    admin_session.assert_icommand("itouch {0}/testobj".format(test_coll))
+                    sleep(5)
+                    # hit the apply_atomic_acl api
+                    self.assertIsNotNone (repeat_until (operator.eq, True) (self.delay_queue_is_empty) (admin_session))
+                    admin_session.assert_icommand("""cd ${{HOME}} ;  python3 ~/scripts/irods/test/atomic_acl_ops.py -v {self.venv_dir} """
+                                                  """ '/{0.zone_name}/home/{0.username}'/{test_coll}/testobj {self.user0.username} write """
+                                                  """ {self.user1.username} read """.format(admin_session, **locals()), use_unsafe_shell=True)
```

Comment:
feels weird to hold it this way...  not sure i have a strong suggestion though...  @alanking ?

Reply:
First, this change will introduce a dependency on PRC, which is currently unprecedented. Personally, I don't think we want that.

HOWEVER, I have learned very recently that no mechanism currently exists which will trigger the atomic API PEPs. So seeing as how this is the only way to do this in existence, I think we can let it pass for now.

We need to do one of two things:

1. The test hook needs to install PRC in order for the tests to run
2. We add a `unittest.skip` decorator to this test with a note about how it can only run on systems with PRC available and is only being run manually at this time?

Let's make an issue for when the magical new `imeta` option appears :)

Reply:
The precedent is there, this was done for` test_indexing_with_atomic_metadata_ops_66` within this same test suite. I don't know, there might have been other reasonable choices, but none occurred.

Reply:
I see, I did not know this. Well, we will need to do one of the two things above regardless in order to test indexing. Do you have a preference for which?

Reply:
Well, I'd rather not compile and link a C++ application just for the purposes of a test, so I think ... PRC.

Reply:
So I guess (1) install in test hook...  Is it a mostly cosmetic issue whether we install it here or in test hook?

Reply:
Well, the two options are:

1. The test hook needs to install PRC in order for the tests to run
2. We add a unittest.skip decorator to this test with a note about how it can only run on systems with PRC available and is only being run manually at this time?

Seeing as how PRC is already required from a previous test, I suspect that the test hook will not succeed in running the tests, so I'm leaning toward option 2.

Keeping the PRC implementation is fine, but we do need to skip the test or make the test hook go.

Reply:
If we go with option 1, I think installing it here or in the test hook should be fine. I think slight preference for test hook

Reply:
making another issue for this, I do not feel I have time for it presently

Reply:
Cannot resolve this conversation, may not have permissions ...